examples/benzeneselenol.cjson

```diff
@@ -0,0 +1,61 @@
+{
+  "chemical json": 0,
+  "name": "Benzeneselenol",
+  "inchi": "1S/C6H6Se/c7-6-4-2-1-3-5-6/h1-5,7H",
+  "formula": "C 6 H 6 Se",
+  "atoms": {
+    "elements": {
+      "number": [  6,  6,  6,  6,  6,  6,  1,  1,  1,  1,  1, 34,  1 ]
+    },
+    "coords": {
+      "3d": [  -2.894762	-0.013642	 -0.001528
+               -2.206251	 1.193851	 -0.002521
+               -0.816426	 1.215302	 -0.002201
+               -0.103352	 0.018392	  0.003106
+               -0.790663	-1.194384	  0.00585
+               -2.179957	-1.205971	  0.001789
+               -3.975843	-0.025301  -0.002904
+               -2.750234	 2.129137	 -0.005276
+               -0.296184	 2.163018	 -0.007326
+               -0.247467	-2.130231	  0.013226
+               -2.702896	-2.153075	  0.003664
+                1.821080  -0.043378	 -0.003876
+                2.004358	 1.410007	  0.103449 ]
+    },
+    "atomcharges": {
```

Comment:
Shouldn't this be something like "charges" or "partialcharges" since it's already in the atom block? I do like recording "mulliken" since there are a wide range of models.